internal/x11/wm/desk.go

```diff
@@ -147,6 +148,7 @@ func NewX11WindowManager(a fyne.App) (fynedesk.WindowManager, error) {
 	go func() {
 		for {
 			<-listener
+			mgr.updateBackgrounds()
 			mgr.refreshBorders()
 			mgr.configureRoots()
 		}
```

Comment:
Same as one of the comments above, if it works :)

Reply:
Actually, I still think this hasn't been completed. Can be tidied up to `for range listener` while at it :)

Reply:
Yup, I missed that there were 4 occurrences over 2 files sorry